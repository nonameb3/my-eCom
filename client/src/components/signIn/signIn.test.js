import React from "react";
import { shallow } from "enzyme";

import SignIn from "./signIn.component";

let wrapper;
const mockDispatch = jest.fn();
jest.mock("react-redux", () => ({
  useDispatch: () => mockDispatch
}));

wrapper = shallow(<SignIn />);
it("should render sign-in component", () => {
  expect(wrapper).toMatchSnapshot();
});
