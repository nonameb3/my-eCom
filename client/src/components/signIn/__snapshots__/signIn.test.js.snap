// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render sign-in component 1`] = `ShallowWrapper {}`;
